lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -161,18 +161,91 @@ def self.detect_bump_type(version_name_a, version_name_b)
         end
       end
 
+      def self.validate_input_if_appending_phc_version?(append_hybrid_common_version, include_prereleases, hybrid_common_version)
+        if append_hybrid_common_version
+          UI.user_error!("Appending the PHC version to prerelease versions violates SemVer.") if include_prereleases
+          UI.user_error!("Cannot append a nil PHC version.") if hybrid_common_version.nil?
+          UI.user_error!("Cannot append a blank PHC version.") if hybrid_common_version.strip.empty?
+        end
+      end
+
+      def self.validate_new_version_if_appending_phc_version?(append_hybrid_common_version, new_version_number, hybrid_common_version)
+        if append_hybrid_common_version
+          if new_version_number.include?("-")
```

Comment:
I've been thinking, would it be cleaner to separate these into a constant? So something like: `PRERELEASE_DELIMITER = "-"`, same for metadata delimiter.

Reply:
Good one! https://github.com/RevenueCat/fastlane-plugin-revenuecat_internal/pull/67/commits/c8e582375131f5e18667369dd5997278e6064f2e